mortar/src/main/java/mortar/PopupPresenter.java

```diff
@@ -31,6 +32,9 @@
 
   private D whatToShow;
 
+  // TODO(ray) If we're going to keep the presenters (NO!) the finder should be set via the
+  // constructor to fix some of our current testing woes.
+  private final BundleService.Finder serviceFinder = new BundleService.Finder();
```

Comment:
Adding a `BundleService.Finder` constructor param to all presenters would be sad.